source/mainnet/net/desktop-wallet/reset-data.rst

```diff
@@ -0,0 +1,42 @@
+.. _reset-data:
+
+==============================
+Reset desktop wallet data
+==============================
+
+All the local data of the desktop wallet is stored in a database in the applications user data folder.
+
+The desktop wallet does not remove your user data when it is uninstalled, to avoid users losing their data, and to simplify the update process of the application.
+
+We will here describe the process of deleting this user data, because it might be necessary to do, but you should make sure that you have a :ref:`backup <export-import>` with your data, before doing so.
```

Comment:
```suggestion
Occasionally, it might be necessary to delete the user data. The following steps describe the process of deleting this user data. Make sure that you have a :ref:`backup <export-import>` with your data before starting this process.
```